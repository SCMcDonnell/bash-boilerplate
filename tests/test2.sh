#!/bin/bash

source ../boilerplate.sh

exit_on_error 'echo "1+2" | bc'
exit_on_error 'asdasdsa asa sgfh jyh'

echo "You should never see this line"

