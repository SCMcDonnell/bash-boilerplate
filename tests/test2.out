3
../boilerplate.sh: line 33: asdasdsa: command not found
